src/types/api.ts

```diff
@@ -0,0 +1,9 @@
+type LogKeeperMetadata = {
```

Comment:
I always thought the `k` was lowercase, but ignore me if it's wrong
```suggestion
type LogkeeperMetadata = {
```